Compilation fails with casacore 2.1.0

```
src/quantamath.cc:72:35: error: member reference base type 'void' is not a structure or union
        unitdesc.append(i->second).getFullName();
        ~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
```

--- a/src/quantamath.cc
+++ b/src/quantamath.cc
@@ -69,7 +69,7 @@ namespace casacore {
                  (CASACORE_MINOR_VERSION>0 || (CASACORE_MINOR_VERSION==0 \
                          && CASACORE_PATCH_VERSION>3)))
         // Use the getter introduced in casacore 2.0.4
-        unitdesc.append(i->second).getFullName();
+        unitdesc.append((i->second).getFullName());
 #else
         // Do the same thing with some string parsing (yuk)
         oss << i->second;

